src/Endpoints/Results/Eligibility.php

```diff
@@ -109,6 +117,16 @@ public function __construct($data = [], $responseCode = null)
         }
     }
 
+    /**
+     * Kind is always 'general' for eligibility at this time
+     *
+     * @return string
+     */
+    public function getKind()
+    {
+        return FeePlan::KIND_GENERAL;
```

Comment:
I think it's easy enough to return the real value from the received data.
The API client might be used in a POS system context and this hardcoded value would become an issue

Reply:
kind is not returned by the API.
This will be an update when API Eligibility EndPoint will be used in POS context (this is not a product request at this time) => YAGNI